nano/node/node.cpp

```diff
@@ -1108,6 +1109,12 @@ void nano::node::block_confirm (std::shared_ptr<nano::block> block_a)
 	}
 }
 
+bool nano::node::block_confirmed (nano::block_hash const & hash_a)
+{
+	auto transaction (store.tx_begin_read ());
+	return store.block_exists (transaction, hash_a) && ledger.block_confirmed (transaction, hash_a);
```

Comment:
I've wondered if `block_account_height` (called inside `block_confirmed`) should just return 0 if the block doesn't exist and not actually error. Would save other places having to check if a block exists (like here)

Reply:
Only found this location and it's currently only used for tests, makes sense but could be handled in another PR, what do you think?

Reply:
sure